wgpu-bindings/bindings.h

```diff
@@ -0,0 +1,112 @@
+#include <cstdint>
+#include <cstdlib>
+
+enum class PowerPreference {
+  Default = 0,
+  LowPower = 1,
+  HighPerformance = 2,
+};
+
+template<typename B>
+struct Buffer;
+
+template<typename B>
+struct CommandBuffer;
+
+template<typename B>
+struct ComputePass;
+
+template<typename B>
+struct Device;
+
+template<typename B>
+struct RenderPass;
+
+template<typename B>
+struct ShaderModule;
+
+struct ShaderModuleDescriptor;
+
+template<typename T>
+struct Handle {
+  T *_0;
+};
+
+using DeviceHandle = Handle<Device<B>>;
```

Comment:
Generics either aren't working here, or they're expected to be resolved manually by us.